generators/generator-botbuilder/generators/app/templates/core/tsconfig.json

```diff
@@ -3,13 +3,15 @@
     "composite": true,
     "declaration": true,
     "target": "es2017",
-    "module": "commonjs",
+    "module": "NodeNext",
```

Comment:
what's the motivation to diverge from commonjs? afaik we have not finalized support to latest node

Reply:
We had to include the `moduleResolution` property to solve a TS issue. It was recommended to match the value for the `module` property with the one for `moduleResolution`, however, it works if we keep the _commonjs_ setting.
![image](https://github.com/user-attachments/assets/bfd1304d-893f-4553-af1d-2f06d3d70278)

Reply:
I'd recommend to keep using commonjs

Reply:
Sorry, I was confused, the value for `moduleResolution` must match the one for `module`. it does not work with commonjs, but _NodeNext_ supports both CommonJS and ECMAScript modules ([documentation](https://www.typescriptlang.org/docs/handbook/modules/reference.html#the-module-compiler-option)).